platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -521,13 +521,15 @@ public void onSurfaceTextureUpdated(SurfaceTexture surface) {
   @CallSuper
   protected void onDetachedFromWindow() {
     super.onDetachedFromWindow();
-    mapZoomButtonController.setVisible(false);
+    if (mapZoomButtonController != null) {
```

Comment:
If we would go this route, we should  move MapboxZoomButtonController to a seperate component.